docs/quality-assurance/test-tooling.md

```diff
@@ -30,3 +32,58 @@
 |Jmeter|Load test functional behaviour and measure performance|https://jmeter.apache.org/||
 |REST-assured|Testing and validating REST services|http://rest-assured.io/||
 |swagger-request-validator|A Java library for validating request/responses against a OpenAPI / Swagger specification|https://bitbucket.org/atlassian/swagger-request-validator||
+
+## Browser Automation - what tools to use?
+This section will cover the browser automation tools we use at the UKHO and the recommended uses for each one. 
+
+The 3 recommended tools for Browser Automation (UI, e2e and Integration) at the UKHO are:
+
+### Cypress
+Cypress is a front end testing tool built for testing web apps. Cypress test code is executed directly within the browser as JavaScript, meaning there are no language or driver bindings and the tests can execute much faster and with more reliability.
+
+Cypress, however, does have its limitations. It has fundamently been created to test your application and your application only, and works best if the Application-Under-Test is a true Single Page Application.
```

Comment:
Not sure about the Single Page Application bit as a multipage application that is all your domain will work just as well.
Worth noting explicitly about cross domain and popups and that some authentication can be worked around but not all